docs: mention making the GitHub App public and APP_INSTALL_URL (#25188)
coder/coder: docs/admin/external-auth/index.md

   ![Install GitHub App](../../images/admin/github-app-install.png)

1. Make the app installable by other users. In the app's **Advanced**

>    tab, click **Make this GitHub App public**.

Reviewer: ```suggestion
   tab, select **Make this GitHub App public**.
```

Nit: I use **select** instead of **click**, to cater to individuals using screen readers or other assistive technology to interact with their computer.

   Without this, every user other than the app owner sees a GitHub 404

>    when they click **Link GitHub** in Coder. Each user must also install

Reviewer: ```suggestion
   Without this, every user other than the app owner gets a GitHub 404
   when they selects **Link GitHub** in Coder. Each user must also install
```

Similar nit about the verb "see".

   the app on their own account before linking. To surface an

>    **Install GitHub App** link in the Coder UI, set:

Reviewer: ```suggestion
   **Install GitHub App** link in the Coder UI, set the following
   environment variable:
```
